app/src/main/res/layout/fragment_home.xml

```diff
@@ -3,4 +3,28 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"/>
+    android:layout_height="match_parent" >
+
+    <EditText
+        android:id="@+id/answer"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginLeft="16dp"
+        android:layout_marginTop="60dp"
+        android:ems="10"
+        android:inputType="textPersonName"
+        android:text="Name"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+        android:id="@+id/btnSubmit"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginLeft="16dp"
+        android:text="Submit"
+        app:layout_constraintBaseline_toBaselineOf="@+id/answer"
+        app:layout_constraintStart_toEndOf="@+id/answer" />
```

Comment:
There were no elements on this page, and based off of the description of the project there will be a lot of input information so I gave something to work with from here. Assuming this is where they will have the quiz for their books, this home page will need to use this format for their questions